mail_broker_telegram/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Dixmit
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mail Telegram Broker",
+    "summary": """
+        Set a broker for telegram""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Dixmit,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/social",
+    "depends": ["mail_broker"],
+    "data": ["views/mail_broker.xml"],
+    "qweb": ["/static/src/xml/thread.xml"],
+    "external_dependencies": {"python": ["python-telegram-bot", "cairosvg", "lottie"]},
```

Comment:
Where cairosvg is used?

Reply:
When processing stickers if I remember properly

Reply:
I don't see the usage. I will test if we can remove it.

Reply:
Actually is a requirement of lottie...

https://gitlab.com/mattbas/python-lottie/-/blob/master/requirements.txt?ref_type=heads#L7